content/spin/v2/quickstart.md

```diff
@@ -420,27 +421,97 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 {{ startTab "Python"}}
 
-Use the `spin new` command and the `http-py` template to scaffold a new Spin application.
+You can install the Spin template for Python HTTP handlers from the [spin-python-sdk repository](https://github.com/fermyon/spin-python-sdk) using the following command:
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ spin new
-Pick a template to start your application with:
-> http-py (HTTP request handler using Python)
-Enter a name for your new application: hello_python
-Description: My first Python Spin application
-HTTP path: /...
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk --update
 ```
 
+The above command will install the `http-py` template and produce an output similar to the following:
+
+<!-- @nocpy -->
+
+```text
+Copying remote template source
+Installing template http-py...
+Installed 1 template(s)
+
++---------------------------------------------+
+| Name      Description                       |
++=============================================+
+| http-py   HTTP request handler using Python |
++---------------------------------------------+
+```
+
+**Please note:** For more information about managing `spin templates`, see the [templates section](./cli-reference#templates) in the Spin Command Line Interface (CLI) documentation.
+
 This command created a directory with the necessary files needed to build and run a Python Spin application.  Change to that directory, and look at the files.  It contains a minimal Python application:
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ cd hello_python
-$ tree
-.
+$ spin new -t http-py hello-python --accept-defaults
+```
+
+Once the component is created, we can change into the `hello-python` directory, create and activate a virtual environment and then install the component's requirements:
+
+<!-- @selectiveCpy -->
+
+```console
+$ cd hello-python
+```
+
+Create a virtual environment directory (we are still inside the Spin app directory):
+
+<!-- @selectiveCpy -->
+
+```console
+# python<version> -m venv <virtual-environment-name>
+$ python3 -m venv venv-dir
+```
+
+Activate the virtual environment (this command depends on which operating system you are using):
+
+<!-- @selectiveCpy -->
+
+```console
+# macOS & Linux command to activate
+$ source venv-dir/bin/activate
+```
```

Comment:
I can't put it as a suggestion because Too Many Backticks but please copy the Windows rubric too?

Reply:
Added, thanks.